bec_alerts/processor.py

```diff
@@ -79,6 +79,9 @@ def stack_frames(self):
 
         return []
 
+    def __repr__(self):
+        return f'<SentryEvent id={self.id} group_id={self.group_id}>'
```

Comment:
`.id` and `.group_id` are both properties that access `self.data`, but do it in a way that could kick up a `KeyError` if things are horribly wrong. Because the world tends towards fire-strewn maelstorm of horribleness and we're often looking at `__repr__` when things are on fire, I would either change the property code to something like:

```
@cached_property
def group_id(self):
    return self.data.get('groupID', 'i hate worms')
```

or do `self.data.get(...)` directly in the `__repr__`. That way even if things are on fire, you still have a chance of gleaning something from error output.

Reply:
The constructor throws an exception if these keys don't exist. This doesn't guard against someone modifying the dict after it is passed to the constructor, though.

I don't think it'll ever happen, but I'll add it with audible grumbling.

Reply:
No worries--that seems fine.